runtime/rust/python-wheel/rust/lib.rs

```diff
@@ -339,7 +339,16 @@ async fn process_stream(
     let mut stream = stream;
     while let Some(response) = stream.next().await {
         // Convert the response to a PyObject using Python's GIL
-        let annotated: RsAnnotated<serde_json::Value> = serde_json::from_value(response).unwrap();
+        // TODO: Remove the clone, but still log the full JSON string on error. But how?
+        let annotated: RsAnnotated<serde_json::Value> = match serde_json::from_value(
+            response.clone(),
+        ) {
+            Ok(a) => a,
+            Err(err) => {
+                log::error!(%err, %response, "process_stream: Failed de-serializing JSON into RsAnnotated");
+                break;
+            }
+        };
```

Comment:
I believe if we remove `%response` from the log, we can avoid the `clone()` - but I'm not sure how much this hurts the logging clarity.
```suggestion
        let annotated: RsAnnotated<serde_json::Value> = match serde_json::from_value(
            response,
        ) {
            Ok(a) => a,
            Err(err) => {
                log::error!(%err, "process_stream: Failed de-serializing JSON into RsAnnotated");
                break;
            }
        };
```